libtiny_tui/src/lib.rs

```diff
@@ -41,8 +41,8 @@ pub struct TUI {
 }
 
 impl TUI {
-    pub fn run(colors: Colors, runtime: &mut Runtime) -> (TUI, mpsc::Receiver<Event>) {
-        let tui = Rc::new(RefCell::new(tui::TUI::new(colors)));
+    pub fn run(colors: Colors, tsmsg: bool, runtime: &mut Runtime) -> (TUI, mpsc::Receiver<Event>) {
+        let tui = Rc::new(RefCell::new(tui::TUI::new(colors, tsmsg)));
```

Comment:
Please rename all `tsmsg`s in the code to `force_ts` (I'm open to naming suggestions if you have any)